packages/api/test/unit/beacon/testData/lightclient.ts

```diff
@@ -57,7 +57,7 @@ export const testData: GenericServerTestCases<Api> = {
     },
   },
   getBootstrap: {
-    args: [toHexString(root)],
+    args: [toHexString(root), "json"],
```

Comment:
You must add unit tests for the ssz variant. Note just setting this `"ssz"` won't actually touch that code path. Double check code is being tested in your approach